app/models/manageiq/providers/ansible_tower/embedded_ansible_worker.rb

```diff
@@ -0,0 +1,20 @@
+class ManageIQ::Providers::AnsibleTower::EmbeddedAnsibleWorker < MiqWorker
```

Comment:
I would resort to `ManageIQ::Providers::EmbeddedAnsible` only as a last option, because it adds a new Provider.

Why not make this a `ManageIQ::Providers::AnsibleTower::TowerWorker` which monitors a tower? First is embedded, next could be external?